README.md

```diff
@@ -66,9 +66,20 @@ followed by a possibly unbounded number of `onNext` signals (as requested by `Su
 #### NOTES
 
 - The specifications below use binding words in capital letters from https://www.ietf.org/rfc/rfc2119.txt
-- The terms `emit`, `signal` or `send` are interchangeable. The specifications below will use `signal`.
-- The terms `synchronously` or `synchronous` refer to executing in the calling `Thread`.
-- The term "return normally" means "only throws exceptions that are explicitly allowed by the rule".
+
+### Glossary
+
+| Term                      | Definition                                                                                             |
+| ------------------------- | ------------------------------------------------------------------------------------------------------ |
+| <a name="term_signal">Signal</a> | As a noun: one of the `onSubscribe`, `onNext`, `onComplete` or `onError` methods. As a verb: calling/invoking a signal. |

```

Comment:
I chose to omit `request` and `cancel` because I felt they landed somewhere between `subscribe` and the `onX`s. I'm OK with adding `request` and `cancel` tho.